Makefile

```diff
@@ -0,0 +1,32 @@
+# Make sure we have ansible_collections/sensu/sensu_go as a prefix. This is
+# ugly as heck, but it works. I suggest to all future developer to treat next
+# few lines as an opportunity to learn a thing or two about GNU make ;)
+collection := $(notdir $(realpath $(CURDIR)      ))
+namespace  := $(notdir $(realpath $(CURDIR)/..   ))
+toplevel   := $(notdir $(realpath $(CURDIR)/../..))
+
+err_msg := Place collection at <WHATEVER>/ansible_collections/sensu/sengo_go
+ifneq (sensu_go,$(collection))
+  $(error $(err_msg)
```

Comment:
We're missing the closing `)` here, resulting in  `Makefile:12: *** unterminated call to function 'error': missing ')'.` in case the collection resides at an invalid path.

Same thing on L12 and L14 as well.